FindOpenStudioSDK.cmake

```diff
@@ -0,0 +1,75 @@
+set(OPENSTUDIO_VERSION_MAJOR 3)
+set(OPENSTUDIO_VERSION_MINOR 0)
+set(OPENSTUDIO_VERSION_PATCH 0)
+set(OPENSTUDIO_VERSION "${OPENSTUDIO_VERSION_MAJOR}.${OPENSTUDIO_VERSION_MINOR}.${OPENSTUDIO_VERSION_PATCH}")
+
+find_package(openstudio "${OPENSTUDIO_VERSION}" CONFIG)
+if(openstudio_FOUND)
+  message("Found specified openstudio at ${openstudio_DIR}")
+else()
+
+  # TODO: currently unsupported, we do not host zip/tar.gz anywhere
+  # This will make it throw, which an informative message
+  find_package(openstudio "${OPENSTUDIO_VERSION}" CONFIG REQUIRED)
```

Comment:
Currently this will throw as it does now, and the following will never be processed.